packages/community-cli-plugin/src/utils/metroPlatformResolver.js

```diff
@@ -29,13 +36,19 @@ export function reactNativePlatformResolver(platformImplementations: {
   [platform: string]: string,
 }): CustomResolver {
   return (context, moduleName, platform) => {
+    let platformOrVariant: string | null =
+      // $FlowFixMe
+      context.customResolverOptions.variant || platform;
```

Comment:
Not sure how to cope with this type, so I disabled the checker. Would love to resolve it properly
<img width="810" alt="image" src="https://github.com/facebook/react-native/assets/5106466/09c73291-b3e9-4a79-99c2-982a01eae47b">